bessctl/conf/perftest/nat.bess

```diff
@@ -41,11 +41,15 @@ ip = scapy.IP(src='10.0.0.1', dst='192.168.1.1')
 udp = scapy.UDP(sport=10001, dport=10002)
 payload = 'helloworld'
 pkt_bytes = bytes(eth/ip/udp/payload)
+nat_config = [{'ext_addr': '1.1.1.1', \
+    'port_ranges': [{'begin': 0, 'end': 65535, 'usable': True}]}, \
```

Comment:
Perhaps it's my personal fetish, but can we replace `usable` with `suspended` (or something like that)? It will reduce verbosity by skipping the argument, since the Protobuf default value `False` can be used here.

Reply:
if `port_ranges` is not specified (thus an empty list), maybe we can assume all ports can be used, rather than none? I see this alternative is less "pure" but makes it much less verbose too.